test-suite/Makefile

```diff
@@ -38,11 +38,11 @@ LIB := ..
 BIN := $(shell cd ..; pwd)/bin/
 COQFLAGS?=
 
-coqc_boot := $(BIN)coqc -coqlib $(LIB) -boot -q -test-mode -R prerequisite TestSuite $(COQFLAGS)
+coqc_boot := $(BIN)coqc -coqlib $(LIB) -q -test-mode -R prerequisite TestSuite $(COQFLAGS)
 coqc := $(BIN)coqc -coqlib $(LIB) -R prerequisite TestSuite $(COQFLAGS)
```

Comment:
These variables are kind of awkward. _boot doesn't mean -boot (which doesn't exist anymore), it means -test-mode. Also why do we allow loading the rc file in the non _boot variable?

Reply:
Yep this RC-file stuff is strange, I'll add `-q`.

Reply:
I'd be great if we could get rid of `-test-mode` indeed.

Reply:
In fact #9588 does improve this too.